### PR TITLE
Mentor Follow go bye-bye

### LINKED ISSUE #127: The follow icon on mhelp messages doesn't work
Basically, you know how theres a "F" next to nearly anything that happens that you can click on and your ghost teleports to it, mhelp messages also have that F, but if you click it nothing actually happens.

### PR DESCRIPTION
**Pull request by Kerbin-Fiber**
## About The Pull Request
Does as the title says.

## Why It's Good For The Game
useless code gone

Closes #127
Also fixes a small grammatical error in a comment. Just cuz.

## Changelog
:cl:
tweak: Mentor Follow has been removed. Never even worked.
/:cl:

### CODE REVIEW

File: code/modules/mentor/mentorhelp.dm [hunk 1]
```diff
@@ -97,6 +97,6 @@
 		. += "*no key*"
 
 	if(include_follow)
-		. += " (<a href='?_src_=mentor;mentor_follow=[REF(M)];[MentorHrefToken(TRUE)]'>F</a>)"
-
+		if(isobserver(M))
```
Comment (ike709) [inline code/modules/mentor/mentorhelp.dm]: Just put this in the above check.

**Review by francinum — CHANGES_REQUESTED** [review]
Just cull the entire follow link.

It doesn't even *work* and it never has.

`/datum/mentor/Topic()` doesn't actually exist.

**Comment by Sinestia** [comment]
it doesnt work for admemes either cull it

**Comment by Rukofamicom** [comment]
This is actually a potentially necessary part of one of my bounties, but I'm sure it can just be re-added in an actually functional way.

I've been trying to find a coder willing to make that link put a mentor in as that player's imaginary friend temporarily, with a return to body function when teaching is done.

**Review by qwertyquerty — DISMISSED** [review]

File: code/modules/mentor/mentorpm.dm [hunk 2]
```diff
@@ -59,21 +59,21 @@
 	var/show_char = CONFIG_GET(flag/mentors_mobname_only)
 	if(C.is_mentor())
 		if(is_mentor())//both are mentors
-			to_chat(C, "<span class='mentorfrom'>Mentor PM from-<b>[key_name_mentor(src, C, 1, 0, 0)]</b>: [msg]</span>")
-			to_chat(src, "<span class='mentorto''>Mentor PM to-<b>[key_name_mentor(C, C, 1, 0, 0)]</b>: [msg]</span>")
+			to_chat(C, "<span class='mentorfrom'>Mentor PM from-<b>[key_name_mentor(src, C, 1, 0)]</b>: [msg]</span>")
+			to_chat(src, "<span class='mentorto''>Mentor PM to-<b>[key_name_mentor(C, C, 1, 0)]</b>: [msg]</span>")
 
 		else		//recipient is an mentor but sender is not
-			to_chat(C, "<span class='mentorfrom'>Reply PM from-<b>[key_name_mentor(src, C, 1, 0, show_char)]</b>: [msg]</span>")
```
Comment (ike709) [inline code/modules/mentor/mentorpm.dm:66]: Looks like you removed the wrong arg here.

**Pull request merged**